packages/inter-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1727,7 +1615,9 @@ test('overdeposit', async t => {
 // Both loans will initially be over collateralized 100%. Alice will withdraw
 // enough of the overage that she'll get caught when prices drop. Bob will be
 // charged interest (twice), which will trigger liquidation.
-test('mutable liquidity triggers and interest', async t => {
+
+// TODO (7047) reinstate when vaults can use auction
```

Comment:
ah. good. so we're keeping track of the tests.

Reply:
yeah, though they migrate to a different file in #7047